src/community/core-periphery.jl

```diff
@@ -44,3 +44,26 @@ function core_periphery_deg end
     c[p[1:kbest]] .= 1
     c
 end
+
+"""
+    terminalnodes( G::SimpleDiGraph )
+
+Given an input SimpleDiGraph, return a dictionary of parent and terminal vertices.
+
+"""
+function terminalnodes( G::SimpleDiGraph )
+    nv = max( length( mission.g.fadjlist), length( mission.g.badjlist ) )
+    #get all neighbors onehop away from each vertex
```

Comment:
accessing `fadjlist` is expressly discouraged. Why not use `nv(g)` to get the number of vertices?

Reply:
that was a copy paste mistake it should be ```G``` sorry :(